common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -243,9 +279,9 @@ function (Sjson, AsyncProcess) {
                 data: data,
                 success: function (sjson) {
                     self.sjson = new Sjson(sjson);
-
-                    var start = self.sjson.getStartTimes(),
-                        captions = self.sjson.getCaptions();
+                    results = self.getBoundedCaptions();
```

Comment:
It looks like we lost the `var` declaration; can you be sure to declare `results`, `start`, and `captions` here?